eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/eventdetails/UidEventDetails.java

```diff
@@ -46,10 +49,11 @@ public UidEventDetails(@NonNull String eventUid)
     @Override
     public void show(@NonNull Activity activity)
     {
-        Intent intent = new Intent(activity, MicroFragmentHostActivity.class);
-        Bundle nestedBundle = new Bundle(1);
-        nestedBundle.putParcelable("MicroFragment", new EventLoaderMicroFragment(mEventUid));
-        intent.putExtra("com.schedjoules.nestedExtras", nestedBundle);
-        activity.startActivity(intent);
+        activity.startActivity(
+                // TODO Use action intent instead?
```

Comment:
No, we don't use Actions "when possible" (I guess it would always be possible). But we use Action Intents to dispatch activity launches (instead of doing that in our code), whenever possible.

Consider the add-to-calendar intent. Instead of checking if the hosting app provides its own activity to handle that and call the default "insert event intent" otherwise, we always fire the add-to-calendar intent and let the system figure out if there is an intent-filter with a higher priority than our default handler.

In this case, there is no need for the hosting app to intercept that call. It's a purely internal thing and there is nothing to dispatch.